src/javascript/app/pages/user/account/settings/professional_client.js

```diff
@@ -28,10 +28,20 @@ const professionalClient = (() => {
         }
 
         const status = State.getResponse('get_account_status.status') || [];
-        if (is_in_page && /professional/.test(status)) {
+        if (is_in_page && status.includes('professional')) {
             $('#loading').remove();
             $('#frm_professional').setVisibility(0);
-            $(`#${/professional_requested/.test(status) ? 'processing' : 'professional'}`).setVisibility(1);
+            $('#professional').setVisibility(1);
+            return;
+        } else if (is_in_page && status.includes('professional_requested')) {
+            $('#loading').remove();
+            $('#frm_professional').setVisibility(0);
+            $('#processing').setVisibility(1);
+            return;
+        } else if (is_in_page && status.includes('professional_rejected')) {
+            $('#loading').remove();
+            $('#frm_professional').setVisibility(0);
+            $('#rejected').setVisibility(1);
```

Comment:
since these three lines are always repeated and the only difference is the id to make visible, why not write a function and pass the id of the element to make visible to it? this will stop us having to repeat code and make changing it in the future easier.

Reply:
@negarn  Where should we organize that function?

Reply:
it can be outside this function, after init function or so